maxim/apis/maxim_apis.py

```
@@ -689,45 +696,205 @@ def get_folders(self) -> List[Folder]:
+                size = 0
+                if content_length:
+                    try:
+                        size = int(content_length)
+                    except ValueError:
+                        size = 0
+                return b"", mime_type, size
+                
+        except Exception as e:
+            raise Exception(f"Failed to download URL attachment {attachment.url}: {str(e)}")
+    
+    def _process_file_attachment(self, attachment) -> tuple[bytes, str, int]:
```

Ruff (0.12.2)

Missing type annotation for function argument `attachment`

(ANN001)